webapp-django/crashstats/crashstats/templates/crashstats/topcrasher.html

```diff
@@ -33,6 +33,9 @@
                   {% endif %}
                 {% endif %}
             </ul>
+            <a href="{{ url('topcrashers:topcrashers') }}?product={{ product }}&amp;version={{ versions[0] }}&amp;date_range_type={{ date_range_type }}&amp;process_type={{ crash_type.replace('all', 'any') }}&amp;platform={{ os_name }}&amp;_facets_size={{ result_count }}&amp;days={{ days }}">
```

Comment:
As far as I know, we have a helper to replace elements of the current query string, but this is building a new URL entirely. I don't see what helper could make that easier.